.github/guides/STANDARDS.md

```diff
@@ -116,7 +116,7 @@ Any sleeping behaviour that you need to perform inside a `SIGNAL_HANDLER` proc m
 Each atom can only register a signal on the same object once, or else you will get a runtime. Overriding signals is usually a bug, but if you are confident that it is not, you can silence this runtime with `override = TRUE`.
 
 ```dm
-RegisterSignal(fork, COMSIG_FORK_STAB, .proc/on_fork_stab, override = TRUE)
+RegisterSignal(fork, COMSIG_FORK_STAB, PROC_REF(on_fork_stab), override = TRUE)
 ```
```

Comment:
Maybe write a bit on which ref you should use?

Reply:
Wait this isnt my repo

Reply:
This should go into RegisterSignal documentation at the very least

Reply:
Added some basic examples/instructions